content/nginxaas-azure/app-protect/configure-waf.md

```diff
@@ -101,7 +101,31 @@ The following table shows the path to the precompiled policy file that needs to
 
 To view the contents of the available security policies, navigate to the azure portal and select the **Security Policies** tab in the App Protect section.
 
-{{<note>}}Custom policies are not supported at this time.{{</note>}}
+## Custom Policies
+
+NGINXaas for Azure also supports custom security policies. You can create and modify custom security policies to deploy to NGINX App Protect Instances using the API or Azure Portal.
+
+### Manage Custom Policies
+
+To create a custom security policy in the Azure Portal:
+
+1. Select your deployment
+2. Select **NGINX app protect WAF** from the menu on the left
+3. Select **Custom Policies**
+4. Select **Add Custom Security Policy** to open the policy editor
+
+In the policy editor, enter the **Name**, **File path**, your policy content, and then select **Save**. The **File path** is optional and will default to the policy **Name** with a ".json" extension if not specified. The **Name** must have a ".json" extension.
```

Comment:
I'm not sure about the wording here.  
> The **Name** must have a ".json" extension
If I actually try this in the portal I get an error with no description.  From what I have observed, **Name** cannot be given an extension.  I'm not sure what the internal API call to use contains, so there might be a bug either on the liftr side or our side.

Reply:
Fixed